rpc/src/v1/types/pubsub.rs

```diff
@@ -30,6 +30,17 @@ pub enum Result {
 	Log(Log),
 	/// Transaction hash
 	TransactionHash(H256),
+	///
+	SyncState(PubSubSyncStatus)
+}
+
+#[derive(Debug, Serialize, Eq, PartialEq, Clone)]
+#[serde(rename_all="camelCase")]
+pub struct PubSubSyncStatus {
+	//	starting_block: U64,
```

Comment:
uncomment or remove and add missing docs

Reply:
ok, as for this, what information do you think we need to include @tomusdrw ?

Reply:
I think just `is_syncing` flag is fine initially, but returning some additional details would be cool as well (if easily available). please coordinate with @axelchalon or @amaurymartiny to check if anything else is required by Fether.